### DISCUSSION TITLE
Simulating Red Cell Shear Stress in Flow

### DISCUSSION THREAD

**Discussion by gsangha1992** [UvaCsl/HemoCell#100] [General discussion]
Hi, 

I saw in a previous thread that it is possible to output fluid shear stress by adding the argument OUTPUT_SHEAR_STRESS to ‘hemocell.setFluidOutputs.’ Is it possible to modify outputs to visualize the changes in red cell shear stress values in a flow simulation (e.g., red blood cells in pipeflow)? 

EDIT: Im specifically interested in how the red cell membrane is deforming to assess in what flow conditions the mechanosensitive ion channel piezo1 might open. Would area force also give me this information?

**Comment by ErenC1** [comment]
Hi,

Everything you need about output is under config/constant_defaults.h.
About shear stress, you already have total force as output. You can use paraview-calculator and visualize it.

Best,
Eren.
